docs/docs/dev_docs/wallets/writing_an_account_contract.md

```diff
@@ -40,13 +40,9 @@ Using the `EntrypointPayload` struct is not mandatory. You can package the instr
 
 Let's go step by step into what the `entrypoint` function is doing:
 
-#include_code entrypoint-init yarn-project/noir-contracts/src/contracts/schnorr_hardcoded_account_contract/src/main.nr rust
-
-We first initialise the private function context using the provided arguments, as we do in any other function. We use a `BoundedVec` container to make it easy to collect the arguments into a single array to be hashed, but we could also have assembled it manually.
```

Comment:
I removed this, because Sean's recent PR hides all of this from the user, so it doesn't need an explanation anymore.